popup-script.js

```diff
@@ -0,0 +1,7 @@
+document.querySelector("#sign_out").addEventListener("click", () => {
+  chrome.runtime.sendMessage({ message: "sign_out" }, function (responese) {
+    if (responese.message === "success") {
+      window.location.replace("./login.html");
```

Comment:
Per my comments elsewhere: your pathing conventions don't seem consistent. I'd like to understand why sometimes the paths are absolute, prefixed etc.